massgov/mayflower
# DP-10441: Add HeaderSlim in Mayflower-react.

## Description
This brings the Header component from EEC over as the new HeaderSlim component. This PR also fixes a bug where the SiteLogo component variable `logo` was pointing to a non-existent path.

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-10441)

## Steps to Test
1.  Pull the branch, run `npm run start`.
2. Browse to Organisms -> HeaderSlim and verify the component looks like the screenshot below.

## Screenshots
![screencapture-localhost-6006-2018-10-03-11_28_27](https://user-images.githubusercontent.com/5257660/46421128-7ebbd800-c6ff-11e8-81d0-ccd3658d3a7b.png)
